### ISSUE TITLE
Problem: Font Awesome icons on report dashboard are different size

### ISSUE THREAD

**Issue by peterVG** [artefactual-labs/AIPscan#141]
The fa-eye icon that toggles the SS API key does not have a size class assigned but the fa-info icon next to the Packages total does. It is set to fa-lg, which looks pretty good and appropriate:

<img width="993" alt="Screen Shot 2021-06-04 at 1 50 43 PM" src="https://user-images.githubusercontent.com/672121/120861373-11aa1180-c53c-11eb-928e-f9e872609a32.png">

Setting the fa-eye icon to match the fa-lg size makes for a more consistent and therefore calming UI effect.

<img width="1022" alt="Screen Shot 2021-06-04 at 1 50 25 PM" src="https://user-images.githubusercontent.com/672121/120861538-4fa73580-c53c-11eb-936c-16f3969ec07b.png">

**Comment by peterVG** [comment]
or now I'm having second thoughts. the eye icon probably looks better in its current "regular" size. I think the Info icon got bumped in size because in this current version of the Font Awesome library its really dark and hard to see the "i" when its smaller.

**Comment by tw4l** [comment]
@peterVG I see what you meant about the smaller size `fa-info-circle` not being so legible. I think it's okay to have them at different sizes but if it irks you and you want to put together a PR to make this more consistent I'm happy to review.

**Comment by peterVG** [comment]
I've played around with options a couple of times now. The eye icon doesn't look right if you make it bigger than default. The info icon is too small and smudged at its default size. Bumping it up one size to fa-lg makes it legible but then slightly mis-aligned in size compared to eye icon. That said, it can't really be avoided. The Info icon without a circle is actually worse as a graphic in this location. And the latter versions of Font Awesome no longer offer an info-circle where the circle fill is transparent making the "i" easier to read. So after all that. Let's just leave as is. Closing.
